sources/ci/set-git-options.md

```diff
@@ -0,0 +1,24 @@
+page_main_title: Setting git options
+main_section: CI
+sub_section: Configuration
+sub_sub_section: Preparing your environment
+page_title: Setting git options
+page_description: How to set options related to git
+
+# Setting git options
```

Comment:
Use two sections. One that shows you how to use depth to get shallow clone, and another for generic git options.